### ISSUE TITLE
Summary: Meets Expectations

### ISSUE THREAD

**Issue by Caealana** [gregajg/intro-data-capstone-biodiversity#5]
I had a lot of fun going over your project, and I was impressed by the amount of detail and work that went into your slides. Your slides were thoughtfully arranged and crafted. You were able to give meaning and life to the numbers and data.

I would suggest more careful curation of what is on your slide. There were some slides that were too cluttered and/or too wordy, and it was hard to know what to focus on. You want to make sure that every little thing presented has a purpose and impact since space is limited. For example:
<img width="859" alt="screenshot at mar 02 11-56-04" src="https://user-images.githubusercontent.com/14001120/36908099-b4f93b76-1e10-11e8-8478-f33c97b88723.png">
While knowing about the three different sheep species is interesting, it is not necessary information for the calculations and tests we ran.

Overall, a great project!

**Comment by gregajg** [comment]
Hi Caealana, thank you for your feedback on the project! I see what you mean on each of the issues and will be aware of that for the future. Do I need to edit and resubmit anything for this?
